bigflow/build/operate.py

```diff
@@ -88,18 +138,44 @@ def build_image(
 
     tag = bf_commons.build_docker_image_tag(project_spec.docker_repository, project_spec.version)
     logger.info("Generated image tag: %s", tag)
-    _build_docker_image(project_spec.project_dir, tag)
+    _build_docker_image(project_spec, tag, cache_params)
 
-    try:
-        _export_docker_image_to_file(tag, image_dir, project_spec.version)
-        dconf_file = Path(project_spec.deployment_config_file)
-        shutil.copyfile(dconf_file, image_dir / dconf_file.name)
-    finally:
-        logger.info("Trying to remove the docker image. Tag: %s, image ID: %s", tag, bf_commons.get_docker_image_id(tag))
+    if export_image_tar:
         try:
```

Comment:
Extracting this if (and else) content to separate functions wouldn't be a good idea?

Reply:
Sure, why not. Refactored.